packages/http-client-python/generator/pygen/codegen/serializers/__init__.py

```diff
@@ -506,8 +465,8 @@ def _additional_folder(self) -> Path:
             return Path("/".join(namespace_config.split(".")[num_of_package_namespace:]))
         return Path("")
 
-    def _serialize_and_write_sample(self, env: Environment, namespace_path: Path):
-        out_path = self._package_root_folder(namespace_path) / Path("generated_samples")
+    def _serialize_and_write_sample(self, env: Environment, namespace: str):
+        out_path = self.exec_path(namespace) / Path("generated_samples")
```

Comment:
i'm wondering how the samples and tests files will be structured? it seems they all put in the corresponding namespace. but it's better to organize them with namespace structure under one `generated_samples`.

Reply:
we always put `generated_samples` and `generated_tests` under root namespace:https://github.com/microsoft/typespec/blob/06f42b6713f6aa4caa55797f9c9355fb2b3529b5/packages/http-client-python/generator/pygen/codegen/serializers/__init__.py#L127-L131